src/commands/get/servers/getservers.go

```diff
@@ -49,8 +50,10 @@ func getServers() error {
 	servers, err := bmcapi.Client.ServersGet(tags)
 
 	if err != nil {
+		log.Error().Err(err)
```

Comment:
I think this will result in the error being logged twice, since the error is shared by cobra as well...